### DISCUSSION TITLE
Enter password dialog window not displaying

### DISCUSSION THREAD

**Discussion by medwatt** [i3/i3#4896] [Q&A]
I have noticed that on my i3 install that the typical window that opens where you type your password when you want to do something requiring root does not show. For example, [here](https://i.imgur.com/tu7O0gl.png)'s when I try to do something in pamac. The window where you type your password does not open and so authentication fails. How do I fix this?

**Comment by Jocadbz** [comment]
This is due to the fact that i3wm doesn't have any authentication agent built-in. To regain this functionality back, you should add:
```exec --no-startup-id /usr/lib64/polkit-gnome/polkit-gnome-authentication-agent-1```
to your i3 config file. Log out and log in again and it should work.

**Comment by morhook** [comment]
You can use lxpolkit by installing that package and then adding on your `$HOME/.config/i3/config`:

```
exec lxpolkit --no-startup-id
```
